src/audio/crossover/crossover.c

```diff
@@ -835,6 +835,11 @@ static int crossover_reset(struct comp_dev *dev)
 
 	crossover_reset_state(cd);
 
+	cd->crossover_process = NULL;
+	cd->crossover_split = NULL;
+	cd->config = NULL;
```

Comment:
this is a config param, we should not be clearing it, this is leaking memory, also can you update the config system to use the new model_handler?

Reply:
Create a PR to fix it: pull #5713